OHHTTPStubs/UnitTests/Test Suites/SwiftHelpersTests.swift

```diff
@@ -454,6 +454,69 @@ class SwiftHelpersTests : XCTestCase {
 #endif
   }
 
+#if swift(>=3.0)
+  func testHasJsonBodyIsTrue() {
+    let jsonStringsAndObjects = [
```

Comment:
Nitpicking, but just realized: why make that an array of tuples, instead of a Dictionary?

Wouldn't change a think in your `for` loop below, but would be more common Swift and would make it order-independent.

Reply:
I'm not sure I agree. Of course I could convert this array to a dictionary but this would mean I can use each JSON sting only once. Unless I've misinterpreted your comment.

```
let dict = [ "{ \"foo\": \"bar\" }", ["foo": "bar" ] ]
```

Reply:
Oh I didn't realize you used some test input strings multiple times, good point